cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -295,11 +303,11 @@
     {% endmacro %}
     {% set js_source = page_js() | trim %}
     {% if js_source | length > 0 %}
-      a.push( '{{ static('js/routes/' + request.path[1:] + 'index.js') }}' );
+      s.push( '{{ static('js/routes/' + request.path[1:] + 'index.js') }}' );
     {% endif %}
 
-    {# Minified dynamic JavaScript loader that injects a script tag in the head of the page #}
-    for(var b,c,d=[],f=document.scripts[0];b=a.shift();)c=document.createElement("script"),"async"in c?(c.async=!1,c.src=b,document.head.appendChild(c)):(d.push(c),c.onreadystatechange=function(){for(var e;d[0]&&"loaded"==d[0].readyState;)e=d.shift(),e.onreadystatechange=null,f.parentNode.insertBefore(e,f)},c.src=b);
+    {# Pass scripts to JavaScript loader. #}
+    jsl(s);
```

Comment:
Minor: should be indented one more level.

Reply:
Fixed